cluster/charts/rook-ceph-cluster/values.yaml

```diff
@@ -0,0 +1,279 @@
+# Default values for a single rook-ceph cluster
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Rook Image
+image: rook/ceph:master
```

Comment:
This is just needed for the toolbox, right? How about moving it under the toolbox section?

Reply:
For every release we will have the same version of rook-ceph and rook-ceph-cluster charts. Can we default this image to the same version as the release? `rook/ceph:<version>`
If we can't use the version as a variable, we can replace it in the build later. We always have a commit before a release to update versions where needed, such as #7961.

Reply:
Changed it to v1.6.3, it seems like you just do a find/replace?

Reply:
If we can't get the version tag from the helm build, let's leave it as `rook/ceph:master` and will do a search/replace later during the release.

Reply:
Ah got it. Yeah I can pull from the chart version.